templates/home.html

```diff
@@ -55,4 +55,5 @@ <h2 class="text-center">{% trans "Featured products" context "Homepage title" %}
       </div>
     </div>
 
+    <script type="application/ld+json">{{ webpage_schema|safe }}</script>
```

Comment:
I think `webpage_schema` may be moved from context_processor to regular function in such a case

Reply:
It could, yes.